components/brave_wallet_ui/page/screens/send/send/send.tsx

```diff
@@ -344,6 +361,43 @@ export const Send = (props: Props) => {
     }
   }, [selectedTokensNetwork])
 
+  React.useEffect(() => {
+    // check if the user has selected an asset
+    if (!contractAddress || !accountAddress || !chainId || selectedSendAsset) return
+
+    const asset = sendAssetOptions.find((option) =>
+      tokenId
+        ? option.contractAddress.toLowerCase() ===
+            contractAddress.toLowerCase() && option.tokenId === tokenId
+        : option.contractAddress.toLowerCase() === contractAddress.toLowerCase()
+    )
+    const account = accounts.find(
+      (account) => account.address === accountAddress
+    )
```

Comment:
can be moved to a memo

Reply:
Fixed here ed6ac7007136978182184c3a54262ac7bc906c7c